shared-ts/utils/dynamo.ts

```diff
@@ -95,3 +101,55 @@ export const getOrganisationsInfo = async (): Promise<Organisations | null> => {
         throw e;
     }
 };
+
+export const getAllOrganisationsInfo = async (): Promise<OrganisationsWithStats | null> => {
```

Comment:
Could we tweak the name of this function to make it slightly more descriptive, as we are grabbing stats as well as info. Something like getAllOrganisationsInfoAndStats?

Reply:
done